lib/controllers/register.js

```diff
@@ -89,6 +89,20 @@ function applyDefaultAccountFields(config,req) {
     req.body.surname = 'Anonymous';
   }
 }
+/**
```

Comment:
I was thinking about that.  My general approach is to not move things into files until I need to require the function by multiple files.  But now I'm wondering if having it in a separate file would encourage others to notice that it's there and to start using it?  Because I do want us to get a control on error handling in this library..

Where would you put it? I image putting it in the `helpers/` folder.